nifi-nar-bundles/nifi-scripting-bundle/pom.xml

```diff
@@ -61,15 +61,33 @@
                 <version>1.10.0-SNAPSHOT</version>
                 <scope>provided</scope>
             </dependency>
+            <dependency>
+                <groupId>org.apache.nifi</groupId>
+                <artifactId>nifi-metrics</artifactId>
+                <version>1.10.0-SNAPSHOT</version>
+                <scope>provided</scope>
+            </dependency>
+            <!--
+                Groovy is provided, by default, in a 'test' scope for use with unit tests
+                Explicitly provide this as 'compile' to provide Groovy support in the
+                scripting processors
+            -->
+            <dependency>
```

Comment:
Similar changes will need to be done to the groovyx bundle.

Reply:
Groovy changes were pulled out into a separate [PR 3547](https://github.com/apache/nifi/pull/3547).